src/bilder/images/edxapp/templates/edxapp/xpro/common_values.yml

```diff
@@ -0,0 +1,577 @@
+DEPRECATED_ADVANCED_COMPONENT_TYPES: []
+DJFS:
+    directory_root: /edx/var/edxapp/django-pyfs/static/django-pyfs
+    type: osfs
+    url_root: /static/django-pyfs
+
+EDXMKTG_USER_INFO_COOKIE_NAME: edx-user-info
+EDX_PLATFORM_REVISION: release
+ELASTIC_SEARCH_CONFIG:
+-   host: elasticsearch.service.consul
+    port: 9200
+    use_ssl: false
+EMAIL_BACKEND: django_ses.SESBackend  # MODIFIED
+# EMAIL_HOST: localhost
+# EMAIL_HOST_PASSWORD: ''
+# EMAIL_HOST_USER: ''
+# EMAIL_PORT: 25
+# EMAIL_USE_TLS: false
+ENABLE_COMPREHENSIVE_THEMING: true  # MODIFIED
+EVENT_TRACKING_SEGMENTIO_EMIT_WHITELIST: []
+EXTRA_MIDDLEWARE_CLASSES: []
+FACEBOOK_API_VERSION: v2.1
+FACEBOOK_APP_ID: FACEBOOK_APP_ID
+FACEBOOK_APP_SECRET: FACEBOOK_APP_SECRET
+FEATURES:
+    AUTH_USE_OPENID_PROVIDER: true
+    AUTOMATIC_AUTH_FOR_TESTING: false
+    CUSTOM_COURSES_EDX: false
+    EMBARGO: true  # ADDED KEY
+    ENABLE_BULK_ENROLLMENT_VIEW: false
+    ENABLE_COMBINED_LOGIN_REGISTRATION: true
+    ENABLE_CORS_HEADERS: true  # MODIFIED
+    ENABLE_COUNTRY_ACCESS: false
+    ENABLE_CREDIT_API: false
+    ENABLE_CREDIT_ELIGIBILITY: false
+    ENABLE_CROSS_DOMAIN_CSRF_COOKIE: true  # MODIFIED
+    ENABLE_CSMH_EXTENDED: true
+    ENABLE_DISCUSSION_HOME_PANEL: true
+    ENABLE_DISCUSSION_SERVICE: true
+    ENABLE_EDXNOTES: false
+    ENABLE_EDX_USERNAME_CHANGER: true  # ADDED KEY
+    ENABLE_ENROLLMENT_RESET: false
+    ENABLE_EXPORT_GIT: true  # MODIFIED
+    ENABLE_GIT_AUTO_EXPORT: true  # ADDED KEY
+    ENABLE_GRADE_DOWNLOADS: true
+    ENABLE_INSTRUCTOR_ANALYTICS: false
+    ENABLE_LTI_PROVIDER: false
+    ENABLE_MKTG_SITE: false
+    ENABLE_MOBILE_REST_API: true
+    ENABLE_OAUTH2_PROVIDER: true
+    ENABLE_PUBLISHER: false
+    ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES: true
+    ENABLE_SPECIAL_EXAMS: true # MODIFIED FOR PROCTORTRACK
+    ENABLE_THIRD_PARTY_AUTH: true
+    ENABLE_UNICODE_USERNAME: true  # ADDED KEY
+    ENABLE_VIDEO_UPLOAD_PIPELINE: true # MODIFIED
+    MILESTONES_APP: true # ADDED KEY FOR PROCTORTRACK
+    PREVIEW_LMS_BASE: {{ key "edxapp/preview-domain" }}  # MODIFIED
+    SHOW_FOOTER_LANGUAGE_SELECTOR: false
+    SHOW_HEADER_LANGUAGE_SELECTOR: false
+    SKIP_EMAIL_VALIDATION: true # ADDED KEY
+    ALLOW_PUBLIC_ACCOUNT_CREATION: true
+    # FROM xPRO CONFIG
+    ALLOW_ALL_ADVANCED_COMPONENTS: true
+    ALLOW_COURSE_STAFF_GRADE_DOWNLOADS: true
+    AUTH_USE_CERTIFICATES: false
+    BYPASS_ACTIVATION_EMAIL_FOR_EXTAUTH: true
+    CERTIFICATES_ENABLED: false
+    DISABLE_LOGIN_BUTTON: false
+    ENABLE_AUTO_COURSE_REGISTRATION: true
+    ENABLE_COURSE_BLOCKS_NAVIGATION_API: true
+    ENABLE_INSTRUCTOR_EMAIL: true
+    ENABLE_INSTRUCTOR_LEGACY_DASHBOARD: true
+    ENABLE_INSTRUCTOR_REMOTE_GRADEBOOK_CONTROLS: true
+    ENABLE_PAID_COURSE_REGISTRATION: false
+    ENABLE_PREREQUISITE_COURSES: true
+    ENABLE_RENDER_XBLOCK_API: true
+    ENABLE_SHOPPING_CART: true
+    ENABLE_SYSADMIN_DASHBOARD: true
+    INDIVIDUAL_DUE_DATES: true
+    LICENSING: true
```

Comment:
actually, should prob be false for xPRO, but doesn't matter too much since it also needs to be set per-course.